vincentvanbot/params.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+IMAGES_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__),'..','raw_data','images'))
+PICKLE_PATH_ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__),'..','raw_data','flat_resized_images'))
```

Comment:
Will we need this after transferring the training data to GCP? There we can access them by bucket id